lib/versioned/^4.0.0/index.js

```diff
@@ -33,16 +34,18 @@ function execute(opts, env, config) {
   logSyncTask(gulpInst, opts);
 
   // This is what actually loads up the gulpfile
-  var exported = require(env.configPath);
+  requireOrImport(env.configPath, function(err, exported) {
+    if (err) {
+      console.error(err);
+      exit(1);
+    }
 
-  registerExports(gulpInst, exported);
+    registerExports(gulpInst, exported);
 
-  // Always unmute stdout after gulpfile is required
-  stdout.unmute();
+    // Always unmute stdout after gulpfile is required
+    stdout.unmute();
 
-  process.nextTick(function() {
```

Comment:
Do you know why this was added? I'm merely guessing that it's no longer required as this code is now in the asynchronous callback to `requireOrImport()`. Would you agree?